### PR TITLE
Fix: larger webapi timeout

### PR DESCRIPTION
**Pull request by jorgemmsilva**
I spent some hours trying to figure out why the webapi was closing http requests (with EOF) on the spam tests.

Turns out that when the node is overloaded, it takes a while to respond to the requests. This PR alleviates this issue by increasing the webapi read/write timeouts to more forgiving values

### CODE REVIEW

File: plugins/webapi/component.go [hunk 1]
```diff
@@ -102,8 +102,8 @@ func provide(c *dig.Container) error {
 			nil,
 			ParamsWebAPI.DebugRequestLoggerEnabled,
 		)
-		e.Server.ReadTimeout = 5 * time.Second
-		e.Server.WriteTimeout = 10 * time.Second
+		e.Server.ReadTimeout = 20 * time.Second
```
Comment (muXxer) [inline plugins/webapi/component.go]: Should we maybe add config parameters for these two?

File: tools/cluster/tests/missing_requests_test.go [hunk 2]
```diff
@@ -52,12 +52,6 @@ func TestMissingRequests(t *testing.T) {
 	err = clu.WaspClient(1).PostOffLedgerRequest(chainID, req)
 	require.NoError(t, err)
 
-	// TODO try to send to only 2 nodes
-	err = clu.WaspClient(2).PostOffLedgerRequest(chainID, req)
```
Comment (muXxer) [inline tools/cluster/tests/missing_requests_test.go:56]: This was removed on purpose, right? :sweat_smile:

Comment (jorgemmsilva) [reply tools/cluster/tests/missing_requests_test.go:56]: yeah, the TODO was to do exactly that :smile:

File: plugins/webapi/component.go (hunk 1 above)
Comment (jorgemmsilva) [reply plugins/webapi/component.go]: sounds reasonable, I'll add it

**Review by dessaya — APPROVED** [review]

**Pull request merged**